### PR TITLE
Safe casting

### PR DESCRIPTION
**Pull request by MikeHathaway**
Addresses PR feedback made by Dmitry about safe casting to protect against overflows: https://github.com/ajna-finance/ecosystem-coordination/pull/44#pullrequestreview-1325454162

### CODE REVIEW

**Comment by MikeHathaway** [comment]
Pipeline is failing due to an issue with the foundry runner. Tests work fine locally

File: src/grants/GrantFund.sol [hunk 1]
```diff
@@ -15,6 +16,8 @@ import { IGrantFund } from "./interfaces/IGrantFund.sol";
 
 contract GrantFund is IGrantFund, ExtraordinaryFunding, StandardFunding {
 
+    using SafeCast for uint;
```
Comment (grandizzy) [inline src/grants/GrantFund.sol]: we don't need this since we do `SafeCast.toUint128` below

**Review by grandizzy — CHANGES_REQUESTED** [review]

**Comment by grandizzy** [comment]
> Pipeline is failing due to an issue with the foundry runner. Tests work fine locally

GrantFund contract size is over deployment limit hence build with sizes if failing

File: src/grants/base/Funding.sol [hunk 2]
```diff
@@ -4,9 +4,12 @@ pragma solidity 0.8.16;
 
 import { Governor }        from "@oz/governance/Governor.sol";
 import { ReentrancyGuard } from "@oz/security/ReentrancyGuard.sol";
+import { SafeCast }        from "@oz/utils/math/SafeCast.sol";
 
 abstract contract Funding is Governor, ReentrancyGuard {
 
+    using SafeCast for uint;
```
Comment (grandizzy) [inline src/grants/base/Funding.sol]: This can be also removed

**Review by grandizzy — CHANGES_REQUESTED** [review]

**Review by grandizzy — APPROVED** [review]
Lgtm

**Pull request merged**